### PR TITLE
Fix/anon in concordance list

### PR DESCRIPTION
**Pull request by tcouch**

### CODE REVIEW

**Comment by tcouch** [comment]
I've converted the concordance_list view to a class-based view called ConcordanceListView as suggested by @raquel-ucl

File: src/rard/research/views/concordance.py [hunk 1]
```diff
@@ -1,41 +1,79 @@
...
+from django.shortcuts import get_object_or_404, render
 from django.utils.decorators import method_decorator
 from django.views.decorators.http import require_POST
-from django.views.generic import ListView
+from django.views.generic import View
 from django.views.generic.edit import CreateView, DeleteView, UpdateView
 
 from rard.research.models import (AnonymousFragment, Concordance, Fragment,
-                                  OriginalText)
+                                  OriginalText, original_text)
```
Comment (raquelalegre) [inline src/rard/research/views/concordance.py]: That's the other one I had spotted that I couldn't find yesterday. What's that `original_text` and how is it different from `OriginalText`?

Comment (tcouch) [reply src/rard/research/views/concordance.py]: I have no memory of adding that in, but I presume I must have. Anyway, I've removed it and all tests still pass.

**Review by raquel-ucl — APPROVED** [review]
Looks good!

**Pull request merged**